src/mixins/pay-api-mixin.ts

```diff
@@ -0,0 +1,36 @@
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { axios } from '@/utils'
+
+// Shared Interfaces
+import { AlterationFilingIF, EffectiveDateTimeIF } from '@/interfaces'
+
+import { FilingCodes } from '@/enums/filingCodes'
+import { CorpTypeCd } from '@/enums'
+/**
+ * Mixin that provides integration with the Auth API.
+ */
+@Component({})
+export default class PayApiMixin extends Vue {
+  /**
+   * Fetches filing fees.
+   * @returns a promise to return the fees for a filing
+   */
+  async fetchFilingFees (filingCode: FilingCodes, entityType: CorpTypeCd, isFutureEffective: boolean): Promise<any> {
```

Comment:
return could be `Promise<FeesIF>` here no? 

Also, just curious. Is this information not something we get through the FeeSummary calculator or is that totally isolated to that component?

Reply:
The FeeSummary component makes the same call to get its information, but I think it would be awkward to scrape that info from it. Better if we don't assume anything about that component.